nixos-vm/vm.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+left-vm () {
+  export SCRIPTPATH=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+  echo building a leftwm virtual machine...
+  echo $SCRIPTPATH
+  cd "$SCRIPTPATH"
+  nixos-rebuild build-vm --flake ../#leftwm
+  ./result/bin/run-leftwm-vm
```

Comment:
Is there a reason to stay in nixos-vm? Or could we `cd -` at the end of the script?

Reply:
the script changes to the nixos-vm directory to have the `result` folder inside `nixos-vm`, and can be `.gitignore`d, but you can either `.gitignore` it both in the root of the project, and inside `nixos-vm` to make sure it's not included

a long way to say "yes, you can", but just giving more context

Reply:
it was a mistake on my part and had the fix locally already, cd - is the correct behaviour. And yeah the reason for it is just to have it dump the result and disk image in the folder regardless of what the current directory is, cleaner this way imo. so you can run the command safely in a folder like leftwm/src and it will get gitignored properly. Could use the root folder aswell but i personally think littering the root with 2 extra build files is a bit ugly.